spyder/plugins/preferences/api.py

```diff
@@ -1,935 +1,802 @@
-# -*- coding: utf-8 -*-

```

Comment:
It seems you changed the line endings of this line and that's why this diff is so big. Please check that.

Reply:
I split the configdialog into different files, so this file is now removed